src/main/java/appeng/tile/misc/CondenserVoidInventory.java

```diff
@@ -52,33 +51,33 @@ public IAEFluidStack injectItems( final IAEFluidStack input, final Actionable mo
 
 		if( input != null )
 		{
-			this.target.addPower( input.getStackSize() / 1000.0 );
+			this.target.addPower( input.getStackSize() / this.channel.transferFactor() );
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Cast one of the operands of this division operation to a "double". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2184)